clang/include/clang/Format/Format.h

```diff
@@ -76,26 +127,16 @@ struct FormatStyle {
     ///       argument2);
     /// \endcode
     BAS_DontAlign,
-    /// Always break after an open bracket, if the parameters don't fit
-    /// on a single line, e.g.:
-    /// \code
-    ///   someLongFunction(
-    ///       argument1, argument2);
-    /// \endcode
+    /// This is **deprecated**. See ``BreakAfterOpenBracketBracedList``,
+    /// ``BreakAfterOpenBracketFunction``, ``BreakAfterOpenBracketIf``,
+    /// ``BreakAfterOpenBracketLoop``, ``BreakAfterOpenBracketSwitch``.
     BAS_AlwaysBreak,
```

Comment:
It's kind of bad, that this is still available, even though it is deprecated. Users of libFormat may just set it and wonder why the behavior changed (i.e it's ignored).
Normally we break the build with deprecations and the users are forced to take action, but they notice.

Reply:
I have deprecated and removed the 4 sub-options `Align`, `DontAlign`, `AlwaysBreak`, and `BlockIndent` in favor of using only `true` and `false`.

Reply:
I guess this would break every build, I don't know if there's a decent way to provide backward compatibility for parsing `Align` and `DontAlign`. Maybe it is better to keep the `enum` intact?